Skills.md

```diff
@@ -4,11 +4,11 @@ https://api.careerbuilder.com/core/tagging/skills
 
 This API supports the HTTP/GET and HTTP/POST methods.  
 
-This API currently supports version 4.1 only. Version is passed in the Accept header as the parameter version (Ex. Accept:application/json;version=4.1). 4.1 contains the latest taxonomy and is more accurate and has a few changes of field names and error formats to conform to standards.
+This API currently supports versions 4.1 and 4.2. Version is passed in the Accept header as the parameter version (Ex. Accept:application/json;version=4.1). 4.2 contains the latest taxonomy and incorporates feedback received about specific terms since the release of V4/4.1 .
 
-| Parameter (4.1) | Required | Description |
+| Parameter (4.1/4.2) | Required | Description |
 |----------------|----------------|-----------------|----------|-------------|
-| version        | Yes | Passed in via the Accept header. Possible values are "4.1".  Defaults to 4.1 | 
+| version        | Yes | Passed in via the Accept header. Possible values are "4.1" or "4.2".  Defaults to 4.1 | 

```

Comment:
Get rid of the default bit please, since this param is required there is no default value